samples/word/20-lists/manage-formatting.yaml

```diff
@@ -0,0 +1,132 @@
+            paragraphs.load("$none"); // No properties needed.
+
+            await context.sync();
+
+            // Start new list using the third paragraph.
+            const list = paragraphs.items[2].startNewList();
+            list.load("$none"); // No properties needed.
+
+            await context.sync();
+
+            // To add new items to the list, use Start or End on the insertLocation parameter.
+            list.insertParagraph("New list item at the start of the list", "Start");
+            const paragraph = list.insertParagraph("New list item at the end of the list (5th level)", "End");
+
+            // Set numbering for list level 1.
+            list.setLevelNumbering(0, Word.ListNumbering.arabic);
+
+            // Set bullet type for list level 5.
+            list.setLevelBullet(4, Word.ListBullet.arrow);
+
+            // Set list level for the last item in this list.
+            paragraph.listItem.level = 4;
+
+            list.load("levelTypes");
+
+            await context.sync();
+
+            const levelTypes = list.levelTypes;
+            console.log("Types of the 9 list levels:");
+            for (let i = 0; i < levelTypes.length; i++) {
+              console.log(`- Level ${i + 1} (index ${i}): ${levelTypes[i]}`);
+            }
+          });
+        }
+
+        async function setup() {
+          await Word.run(async (context) => {
+            const body = context.document.body;
+            body.clear();
+            body.insertParagraph(
+              "Themes and styles also help keep your document coordinated. When you click design and choose a new Theme, the pictures, charts, and SmartArt graphics change to match your new theme. When you apply styles, your headings change to match the new theme. ",
+              "Start"
+            );
+            body.insertParagraph(
+              "Save time in Word with new buttons that show up where you need them. To change the way a picture fits in your document, click it and a button for layout options appears next to it. When you work on a table, click where you want to add a row or a column, and then click the plus sign. ",
+              "Start"
+            );
+            body.insertParagraph(
+              "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
+              "Start"
+            );
+            body.paragraphs
+              .getLast()
+              .insertText(
+                "To make your document look professionally produced, Word provides header, footer, cover page, and text box designs that complement each other. For example, you can add a matching cover page, header, and sidebar. Click Insert and then choose the elements you want from the different galleries. ",
+                "Replace"
+              );
+          });
+        }
```

Comment:
There's a lot of sample paragraphs. Consider streamlining the sample a bit. Does it need text other than the list?